book/A-git-in-other-environments/sections/eclipse.asc

```diff
@@ -1,10 +1,10 @@
 === Git in Eclipse
 
 (((Eclipse)))
-Eclipse ships with a plugin called Egit, which provides a fairly-complete interface to Git operations.
-It's accessed by switching to the Git Perspective (Window > Open Perspective > Other…, and select "Git").
+Eclipse wird mit einem Plugin namens EGit ausgeliefert, das eine ziemlich vollständige Oberfläche für Git-Operationen bietet.
+Der Zugriff erfolgt durch Wechseln in die Git-Perspektive (Window > Open Perspective > Other… und „Git" selektieren).
```

Comment:
Vorschlag: `Der Zugriff erfolgt durch Wechseln in die Git-Perspektive (Fenster > Perspektive öffnen > Andere,  dann "Git" auswählen).`
Vorsicht, Anführungsstrich von oben nach unten gerutscht.

Reply:
Heisst das in Eclipse so, hast du das geprüft? Ich selber habe kein Eclipse installiert, da ich mit der PyCharm IDE arbeite. :-)
Bezüglich Anführungsstriche: Das ist eigentlich so gewollt. Im deutschen sind die vorne unten.

Reply:
Hab ich auch nicht verifiziert, dann bleiben wir bei deiner Version.
Dann bleibt nur noch mein Vorschlag selektieren->auswählen. Gefällt mir deutlich besser, da ich 'selektieren' sehr selten höre/nutze. Was meinst du?

Reply:
Ja, auswählen ist viel besser, da hast du recht. 
Ich habe es gerade in einem PR mit eingearbeitet.

Jetzt hoffe ich, dass ab morgen die HTML Seite des Buches auf Deutsch wieder funktioniert, nachdem ich sie gestern zerschossen habe. :-)

Reply:
Super, danke!
HTML sieht wieder gut aus :D

Reply:
```suggestion
Der Zugriff erfolgt durch Wechseln in die Git-Perspektive (Window > Open Perspective > Other… und „Git“ selektieren).
```
Das rechte Anführungszeichen war falsch